projects/canopy/src/lib/card/card.component.ts

```diff
@@ -5,28 +5,46 @@ import {
   HostBinding,
   QueryList,
   ViewEncapsulation,
+  ElementRef,
 } from '@angular/core';
 
 import { LgCardContentComponent } from './card-content/card-content.component';
 import { LgCardHeaderComponent } from './card-header/card-header.component';
+import { LgCardBottomAreaComponent } from './card-bottom-area/card-bottom-area.component';
+import { LgCardTopAreaComponent } from './card-top-area/card-top-area.component';
 
 @Component({
-  selector: 'lg-card',
+  selector: 'lg-card, lg-card-form-journey',
```

Comment:
I think if we are to provide a form-journey variant we would probably do it with a variant property instead of a distinct component, however it may be that we can achieve it already with the grid and margin/padding directives.